src/Evolution/Systems/Cce/WorldtubeInterfaceManager.cpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/Cce/WorldtubeInterfaceManager.hpp"
+
+#include <deque>
+#include <memory>
+#include <tuple>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Time/TimeStepId.hpp"
+
+namespace Cce {
+
+std::unique_ptr<GHWorldtubeInterfaceManager>
+GHLockstepInterfaceManager::get_clone() const noexcept {
+  auto clone = std::make_unique<GHLockstepInterfaceManager>();
```

Comment:
you should be able to just return a `std::make_unique<GHLockstepInterfaceManger>(*this)`